Makefile

```diff
@@ -141,7 +141,7 @@ proto:
 build: build-manager
 
 build-manager: generate fmt vet lint vendor ## Build manager binary.
-	go build -o bin/manager cmd/manager/manager.go
+	go build -ldflags "-X main.version=$(shell git describe --tags)" -o bin/manager cmd/manager/manager.go
```

Comment:
Could you also add the `ldflags` to the `run-local` target? Besides, also to the Dockerfile. You can pass the arguments with an `ARG` statement.